models/activities/action.go

```diff
@@ -215,7 +215,7 @@ func (a *Action) ShortRepoPath() string {
 // GetRepoLink returns relative link to action repository.
 func (a *Action) GetRepoLink() string {
 	// path.Join will skip empty strings
-	return path.Join(setting.AppSubURL, "/", url.PathEscape(a.GetRepoUserName()), url.PathEscape(a.GetRepoName()))
+	return setting.AppURL + url.PathEscape(a.GetRepoUserName()) + "/" + url.PathEscape(a.GetRepoName())
```

Comment:
The comment said: "GetRepoLink returns relative link to action repository."

I think the behavior should not be changed. Absolute URLs in templates should be avoided. See https://github.com/go-gitea/gitea/issues/19345

Reply:
Ah, didn't know about that.